rules/javascript/lang/non_literal_fs_filename.yml

```diff
@@ -0,0 +1,148 @@
+patterns:
+  - pattern: |
+      $<FS>.$<METHOD>($<...>$<INPUT>$<...>)
+    filters:
+      - variable: FS
+        regex: (?i)\Afs\z
```

Comment:
You should probably use this
https://github.com/Bearer/bearer-rules/pull/293/files#diff-87b2a91bab7a6339a59ea4daaa58401caa3c03d7faa71793a63e423b53b20de5R8